### PR TITLE
Add readme for the multi selection component

### PR DESCRIPTION
**Pull request by JustinyAhin**
## Description
Added documentation for the multi selector inspector component (see #22891)

## How has this been tested?
N/A, documentation only

## Types of changes
Documentation

## Checklist:
- [n/a] My code is tested.
- [n/a] My code follows the WordPress code style. 
- [n/a] My code follows the accessibility standards. 
- [n/a] My code has proper inline documentation. 
- [x] I've included developer documentation if appropriate. 
- [x] I've updated all React Native files affected by any refactorings/renamings in this PR.

### CODE REVIEW

**Comment by JustinyAhin** [comment]
@mkaz for the code sample, I'm wondering if I should rather do something like:

```
const SelectedBlockCount = getSelectedBlockCount();
if ( SelectedBlockCount > 1 ) {
    const MyMultiSelectionInspector = () => <MultiSelectionInspector />;
}
```

Maybe looks more clear?

File: packages/block-editor/src/components/multi-selection-inspector/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+## Table of contents
+
+1. [Development guidelines](#development-guidelines)
+2. [Related components](#related-components)
+
+## Development guidelines
+
+### Usage
+
+Renders an block alignment toolbar with alignments options.
```
Comment (mkaz) [inline packages/block-editor/src/components/multi-selection-inspector/README.md:18]: I think this needs updating, probably from your template

**Review by mkaz — CHANGES_REQUESTED** [review]
I think you missed a line to replace from your template.

Also your code example in the comment is better, maybe prefix with a description of "an example detecting multiple blocks to determine the component used" or something like that.

File: packages/block-editor/src/components/multi-selection-inspector/README.md (hunk 1 above)
Comment (JustinyAhin) [reply packages/block-editor/src/components/multi-selection-inspector/README.md:18]: Oh yup. My bad.

**Review by mkaz — APPROVED** [review]
:+1:

**Pull request merged**